active_storage_base64.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.7.0'
 
   # Dependencies
-  s.add_dependency 'rails', '>= 7.0'
+  s.add_dependency 'rails', '>= 7.1'
```

Comment:
could you accept the new Rails version [7.1.0.beta1](https://rubygems.org/gems/rails/versions/7.1.0.beta1) please ? :)

Reply:
@cprodhomme done, version 3.0.0 released. Let me know if it works for you. Thanks